mav_trajectory_generation/include/mav_trajectory_generation/polynomial.h

```diff
@@ -42,7 +42,8 @@ class Polynomial {
 
   // Maximum degree of a polynomial for which the static derivative (basis
   // coefficient) matrix should be evaluated for.
-  static constexpr int kMaxN = 12;
+  // kMaxN = max. convolution size for N = 12.
+  static constexpr int kMaxN = 12 + 11 - 1;
```

Comment:
Don't like this variable name, since this is suddenly not N as it's used everywhere else.
kMaxConvolutionSize would be fine, and then based on kMaxN.